app/uk/gov/hmrc/cgtpropertydisposals/models/des/returns/DesReturnDetails.scala

```diff
@@ -61,8 +63,14 @@ object DesReturnDetails {
     )
   }
 
-  private def getSource(submitReturnRequest: SubmitReturnRequest): String =
-    submitReturnRequest.agentReferenceNumber.fold("self digital")(_ => "agent digital")
+  private def getSource(submitReturnRequest: SubmitReturnRequest): String = {
+    val source =
+      if (submitReturnRequest.isFurtherReturn)
+        s"${"self digital " + Instant.now.getEpochSecond.toString}"
+      else
+        "self digital"
+    submitReturnRequest.agentReferenceNumber.fold(source)(_ => "agent digital")
+  }
```

Comment:
does "agent digital" need to have this timestamp added?

Reply:
yeah, we'll need it if an agent did the return as well